cclib/method/population.py

```diff
@@ -11,17 +11,19 @@
 
 import numpy
 
-from cclib.method.calculationmethod import Method
+from cclib.method.calculationmethod import Method, MissingAttributeError
 
 
 class Population(Method):
     """An abstract base class for population-type methods."""
 
+    required_attrs = ('homos', 'mocoeffs', 'nbasis')
+    overlap_attributes = ("aooverlaps", "fooverlaps")

```

Comment:
Any reason you switched from single quotes to double quotes? (No real preference, just was surprised.)

Reply:
Done in 83c0738ef18535f16e807a1f5235329e54cf6a3b.